### ISSUE TITLE
Visualization with S3DIS

### ISSUE THREAD

**Issue by banciuadrian** [torch-points3d/torch-points3d#763]
Hello!

From this library I am using the re-implementation of the `S3DIS` loader that loads the data in full. I am doing this for the purpose of visualization, the problem is as follows:

Loading the data with `S3DIS1x1` compared to the `S3DISOriginalFused` results in a different number of points. I am using Area 5 only.
With Fused I get 78649818 points (the number of points presented in the study) and with `S3DIS1x1` I get about 4096*6852 points (which is pre-processed as far as I'm aware), which is about 50 million less. I want to visualize the predicted semantic segmentation labels of my model, but I can't make a one to one correspondence due to the `Fused` version having so many more points.

In this case, how do I go about visualizing the prediction from my model on the whole area that has been trained using the PyTorch Geometric S3DIS dataset? (ideally I want to load the fused area 5 fully and see the predictions on that, i.e., reduce it to the 28 million dimension and have the positions be spatially accurate, which is what I was trying to do by using the Fused "loader")

I suppose the whole question boils down to what process was used for the S3DIS1x1 variant to downsample it. 

Thank you a lot!

**Comment by CCInc** [comment]
Hi! The S3DIS1x1 dataset comes directly from pytorch geometric, you can see information about it [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html#torch_geometric.datasets.S3DIS). 

What I would suggest is doing your training on the Fused dataset. If you use any of the [dataset configurations](https://github.com/torch-points3d/torch-points3d/tree/master/conf/data/segmentation) we provide, the dataset should be automatically voxelized at 4cm (see [here](https://github.com/torch-points3d/torch-points3d/blob/297f0e0906d99185216716144600d793f9fa16fb/conf/data/segmentation/s3disfused-sparse.yaml#L11-L13)). If you need, you can downsample it further. 

Then, when running testing/inference,  there is code built in to upsample the results back to the full, non-voxelized fused dataset using knn upsampling. You can enable this behavior [here](https://github.com/torch-points3d/torch-points3d/blob/297f0e0906d99185216716144600d793f9fa16fb/conf/config.yaml#L23) by setting `tracker_options.full_res=True`. 

Let me know if that helps!
